Makefile

```diff
@@ -327,7 +327,7 @@ _lint:
 	pre-commit run --all-files
 
 test:
-	skipper make $(SKIPPER_PARAMS) _test
+	NUM_WORKERS=2 skipper make $(SKIPPER_PARAMS) _test
```

Comment:
@slaviered is this what you meant?

Reply:
maybe better to move it to conftest.py? and do something like this max(2, os.get(NUM_WORKERS)) ? to be able to add more hosts?